lib/phoenix_live_view/async.ex

```diff
@@ -312,6 +372,29 @@ defmodule Phoenix.LiveView.Async do
     end
   end
 
+  defp handle_kind(socket, _maybe_component, :stream, key, result) do
+    case result do
+      {:ok, {:ok, %{stream: stream, opts: opts}}} ->
+        socket
+        |> Phoenix.Component.assign(key, AsyncResult.ok(get_current_async!(socket, key), nil))
```

Comment:
I wonder if there is something better than `nil`, since if `@foo.ok` or whatever one would use to read would still fail? Perhaps `true`?